python-threatexchange/threatexchange/fetcher/fetch_api.py

```diff
@@ -78,6 +81,8 @@ def get_config_class(cls) -> t.Type[CollaborationConfigBase]:
         """Returns the dataclass used to store records for this API"""
         return CollaborationConfigBase
 
+    # TODO - this doens't work for StopNCII which sends the owner as a string
+    #        maybe this should take the full metadata?
```

Comment:
why not have owner id as an str in the first place, does the break the other API's impl?

Reply:
I think I'm trending in that direction, but I haven't decided yet if this leads to weird results on ThreatExchange, where we currently aren't resolving the owner ids. I'm planning on tackling this in a followup, and I'll create a note in https://github.com/facebook/ThreatExchange/projects/3